Makefile

```diff
@@ -29,14 +29,6 @@ export WAIT_TIMEOUT := 60s
 # Install default ClusterCatalogs
 export INSTALL_DEFAULT_CATALOGS := true
 
-# By default setup-envtest will write to $XDG_DATA_HOME, or $HOME/.local/share if that is not defined.
-# If $HOME is not set, we need to specify a binary directory to prevent an error in setup-envtest.
-# Useful for some CI/CD environments that set neither $XDG_DATA_HOME nor $HOME.
-SETUP_ENVTEST_BIN_DIR_OVERRIDE=
-ifeq ($(shell [[ $$HOME == "" || $$HOME == "/" ]] && [[ $$XDG_DATA_HOME == "" ]] && echo true ), true)
-	SETUP_ENVTEST_BIN_DIR_OVERRIDE += --bin-dir /tmp/envtest-binaries
```

Comment:
We are updating the setup to use the project's bin directory (used already for the other staff), ensuring we don’t interfere with contributors' local development environments."

Reply:
iirc, the reason this is here is because Red Hat reported issues when they ran CI because of quirks of how their CI system sets up the environment. We would be good stewards if we validated changes in this area in RH's CI system before we make them here.

Reply:
Note the conditional: in order for this override to be used we need both:
- $HOME=="" or $HOME=="/"
- XDG_DATA_HOME==""

That will not happen in a developer's local environment.

Reply:
My whole idea is for us 
in local/dev is to have the bins inside of the project
So that I can config the ENVTEST properly and debug the tests with IDE
I will give a look

I do not want either the targets installing staff in my go src or any other place outside of the project.

Reply:
@joelanford Is not only used because the default path will be /usr/local/kubebuilder/, which does not work in the GitHub/ci. If we move to the project's bin dir, would the issues be reported differently?

Reply:
Unresolving because I have a related question.

WIth `--bin-dir` set to a local directory - do we need a conditional for `/tmp/envtest-binaries`?

Reply:
@m1kola 

Is that make sense ?
Can we close this one?